dashboard/src/actions/kube.tsx

```diff
@@ -51,7 +56,11 @@ export function getResource(
       return;
     }
 
-    dispatch(requestResource(key));
+    // If it's not the first request, we can skip the request REDUX event
+    // to avoid the loading animation
+    if (shouldDispatchRequest !== false) {
```

Comment:
Why not `if (shouldDispatchRequest) {` ? Ah, perhaps existing call-sites haven't been updated so get a default of `undefined` which isn't true? Not sure, but wonder if `polling` would be a better arg, which is fine defaulting to false/undefined, so here can be `if(!polling) {...}`. Anyway, fine either way, just found `shouldDispatchRequest` a little confusing at first because it implies it's talking about whether the actual request (to the server) will be dispatched, but it's not, it's just the redux state.

Reply:
> Why not `if (shouldDispatchRequest) {` ? Ah, perhaps existing call-sites haven't been updated so get a default of `undefined` which isn't true?

yes, it's because of that

> Not sure, but wonder if polling would be a better arg, which is fine defaulting to false/undefined, so here can be if(!polling) {...}. Anyway, fine either way, just found shouldDispatchRequest a little confusing at first because it implies it's talking about whether the actual request (to the server) will be dispatched, but it's not, it's just the redux state.

I think I don't understand what `polling` would mean in this context? I am okay changing the var name though.

Reply:
It wasn't so much about changing the name as opposed to choosing something such that the default value could be undefined/false, so that you could just do `if (!polling) {...}` (and perhaps be a little clearer). But totally fine as is, imo.